Added organization and account self deletion (#100)
Axway/platform-management-docs: content/en/docs/management_guide/managing_accounts.md

* Select your organization account theme (Light, Dark, or Match OS)
* Expand or minimize side-bar menus
* Show or hide getting started content on Platform home
* Select your org login rule
* Configure your inactivity logout timeframe
* Enable or disable restoring the previous page on your next login
* Delete your account

> ![Settings page](/Images/settings_tab.png)

Reviewer: Good catch.

You can enable or disable if you want to be redirected to the page previously viewed once signed back in after being signed out due to inactivity in the Platform. This setting is disabled by default (do not restore previous page on next login).

### Delete account

> The option to delete your account is available for the user if there are no paid subscriptions or where the invoicing type is not a contract or license. For example, users who sign up and create an organization with no paid subscriptions will see the delete option. If the user is associated with an organization that has a support access code, you must email support@axway.com or visit [Axway Support](https://support.axway.com/) to request Axway to delete the account. If the user deleting the account is the only member of the organization being deleted, then the associated organization is also deleted.

Reviewer: ```suggestion
The option to delete your account is available for the user if they are not a member of one ore more organizations with paid subscriptions. Users and organization created by sign-up with no subscriptions or trial only subscriptions will see the delete option. A user which is a member of one or more organizations which have a Support Access Code must email support@axway.com or visit [Axway Support](https://support.axway.com/) to request Axway delete the account on their behalf. If the user deleting their account is the only member of their organization, then their organization will also be deleted.
```

![Deleting your account](/Images/account_settings_delete.png)

> 1. Click **Delete**. A modal displays.

Reviewer: ```suggestion
1. Click **Delete**. A confirmation modal displays.
```

> 2. Type the password of your account used to sign into the Platform in the modal.

Reviewer: ```suggestion
2. Enter the password of your account used to sign into the Platform.
```
Redundant. You're already "in the modal".

3. Click **Send code**. An authorization code is emailed to your Platform user account email address.

> 4. Access your email with the authorization code, and then copy and paste or type the code into the **Authorization code** field in the modal.

Reviewer: ```suggestion
4. Copy and paste or type the authorization code received in the email into the **Authorization code** field.
```
